### ISSUE TITLE
Support for CSS transitions

### ISSUE THREAD

**Issue by pmdarrow** [dockyard/ember-wuphf#7]
In the blog post about WUPHF, there is a [JS Bin](http://jsbin.com/luhoquxi/7/edit?css,js,output) that includes support for fading out messages when they're dismissed. I was wondering why you stripped it out when making it an Ember-CLI add-on and if you'd consider adding it back in?

**Comment by bcardarella** [comment]
The css transitions were twitter bootstrap specific. I wanted to have the library be as agnostic as possible.

**Comment by pmdarrow** [comment]
Makes sense. How would you approach adding this back in? Ideally I could do it in a way that could be contributed back to your project as an optional add-on.

**Comment by bcardarella** [comment]
I don't think there is any clean way to do it. You'll have to overwrite the templates, override the `didInsertElement` for the component. It might be cleaner with HTMLBars but I haven't looked into it.

**Comment by bcardarella** [comment]
@pmdarrow I am open to any suggestions on how to improve the flexibility of the library :)
